.github/ISSUE_TEMPLATE/how-to-question.md

```diff
@@ -1,13 +1,19 @@
 ---
 name: How-to question
 about: To ask questions about how to use office.js or the Office developer platform,
-  please post your question on https://stackoverflow.com and tag it with office-js
+  please post your question on Microsoft Q&A
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
->>> PLEASE POST QUESTIONS ON STACK OVERFLOW <<<
+## Please post how-to questions about your code on Microsoft Q&A
 
-Tag your question with office-js or outlook-web-addins
+We have several forums where you can ask the developer community questions about your code. Try one of the following forums:
+- [Technical questions about developing Office Add-ins for Word, Excel, PowerPoint, Outlook, Project, Visio, and OneNote](https://docs.microsoft.com/en-us/answers/topics/office-addins-dev.html)
+- [Technical questions about Access programming in Office](https://docs.microsoft.com/en-us/answers/topics/office-access-dev.html)
+- [Technical questions about developing COM or VSTO Add-ins for Office](https://docs.microsoft.com/en-us/answers/topics/office-vsto-com-dev.html)
+- [Technical questions about the Office JavaScript API](https://docs.microsoft.com/en-us/answers/topics/office-js-dev.html)
+- [Technical questions about SharePoint development and OneDrive development](https://docs.microsoft.com/en-us/answers/topics/sharepoint-dev.html)
+- [Technical questions about deploying and publishing Office or SharePoint apps to AppSource](https://docs.microsoft.com/en-us/answers/topics/microsoft-365-apps-publishing-dev.html)
```

Comment:
```suggestion
We have several forums where you can ask the developer community questions about your code. Try one of the following forums to ask technical questions about:
- [Developing Office Add-ins for Word, Excel, PowerPoint, Outlook, Project, Visio, and OneNote.](https://docs.microsoft.com/answers/topics/office-addins-dev.html)
- [Access programming in Office.](https://docs.microsoft.com/nswers/topics/office-access-dev.html)
- [Developing COM or VSTO Add-ins for Office.](https://docs.microsoft.com/answers/topics/office-vsto-com-dev.html)
- [Office JavaScript API.](https://docs.microsoft.com/answers/topics/office-js-dev.html)
- [SharePoint and OneDrive development.](https://docs.microsoft.com/answers/topics/sharepoint-dev.html)
- [Deploying and publishing Office or SharePoint apps to AppSource.](https://docs.microsoft.com/answers/topics/microsoft-365-apps-publishing-dev.html)
```

Reply:
These are the exact descriptions from the Q&A forums. I think it's better for SEO if we keep them the same as the target.

Reply:
Then at the very least we should remove the "en-us" from the links.

Reply:
You are absolutely right! I'll fix the other anomolies, too. I did that at 1 AM this morning and it sure shows! :-/

Reply:
Actually, on second thought, I do like your ideas of reducing the repetive "Technical questions about." Better for accessibility while still maintaining SEO. Thanks for the awesome feedback.